### PR TITLE
#78 Creating 3D Axis For Plotting Data Points

### LINKED ISSUE #78: Create Axis for 3D graph
# Requirements
The axis must be able to scale on input. By this I mean that the numbers on an axis *MUST* change depending on the range of values given, the highest point being 10 units of scale above the largest data point and lowest point being 10 below the smallest. 

----
 Dev member must keep in mind that this component must be used eventually with other components for the graph so keep compatibility in mind. 

## Nice to have 
The ability to change the field read from a csv file that the axis is representing.

### PR DESCRIPTION
**Pull request by qpb948**
Closes #78

# What was the issue
We needed a 3D axis to plot data points onto so we can visualize the data.  This axis needed to scale based on input as well, so the numbers on the axis will adjust based on a minimum and maximum value that will be read for the CSV (currently just hardcoded values). This PR includes the 3D axis that will be used.

# What was done and why
I created 3 new files: axis.tsx, singleAxis.tsx, and createTIcks.tsx.
axis.tsx calls the component in singleAxis.tsx 3 times, one for each axis (x, y, z).
singleAxis.tsx calculates the range of values that will be displayed on the axis, as well as creating the axis depending on input from axis.tsx.
createTicks.tsx this file creates and positions the ticks and labels correctly on each axis.

# How it was tested
Testing in the WebXR emulator. I put in different values for minValue and maxValue in App.tsx to ensure functionality of scaling works as required.

### CODE REVIEW

**Review by Jesster2829 — APPROVED** [review]
Looks good to me! Thank you for adding in the documentation

File: my-webxr-app/src/Components/axis.tsx
Comment (matthew-buglass) [inline my-webxr-app/src/Components/axis.tsx:1]: Need some unit tests.

File: my-webxr-app/src/Components/createTicks.tsx
Comment (matthew-buglass) [inline my-webxr-app/src/Components/createTicks.tsx:1]: Need some unit tests.

File: my-webxr-app/src/Components/singleAxis.tsx
Comment (matthew-buglass) [inline my-webxr-app/src/Components/singleAxis.tsx:1]: Need some unit tests.

**Review by matthew-buglass — CHANGES_REQUESTED** [review]
Looks good to me. Just need to put in some unit tests.

**Comment by matthew-buglass** [comment]
Also, make sure the "Components" directory gets renamed to "components"

**Pull request merged**